perfmetrics/scripts/fio_logger.py

```diff
@@ -0,0 +1,25 @@
+"""This script takes the logging directory path as input and 
+stores the output result of fio load tests in the directory along with 
+removing log files in the directory which are older than last 10 days.
+Usage:
+python3 fio_logger.py $PATH_TO_LOG_DIR $FIO_OUTPUT_FILE
+"""
+
+import os
+from datetime import date
+import typing
+import sys
+
+def log_fio_output(logging_dir: str, output_file: str):
```

Comment:
the formatting of code is not proper in both the files. Please fix it by referring to existing files under this directory.

Reply:
Done